_pages/keyboards/georgi.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Georgi!"
+layout: qmk
+permalink: /keyboards/georgi/
+redirect_from: 
```

Comment:
You don't actually need Lines 5-6, *unless* you have documentation that directs people to `qmk.fm/georgi/`.

Reply:
Which I do :(

Reply:
Which is fine. This was more "FYI" than "If you include this in the PR, I'll hate you for a week." :smile: